Formula/xgboost.rb

```diff
@@ -0,0 +1,30 @@
+class Xgboost < Formula
+  desc "Scalable, Portable and Distributed Gradient Boosting Library"
+  homepage "https://xgboost.ai/"
+  url "https://github.com/dmlc/xgboost.git",
+      :tag      => "v0.90",
+      :revision => "515f5f5c4779ff5361dcd796e22d55937e1048ea"
+
+  depends_on "cmake" => :build
+  depends_on "gcc@8"
```

Comment:
Sorry to disagree with @javian there, but unless multithreading is essential to the app's behaviour, we generally avoid it. GCC is a heavy dependency, so we want to avoid it if at all possible, and compile with the system compiler.

Using `libomp` maybe help provide OpenMP support, if that's what you need.

And in any case, compiling with an older version of GCC is a definite “no way”.

Reply:
Hey @fxcoudert, thanks for the feedback on this as well. It looks @hcho3 has it working with `libomp` but it isn't merged yet, so will see what the maintainers want to do: https://github.com/dmlc/xgboost/issues/4477